### PR TITLE
Home style

### PR DESCRIPTION
**Pull request by SathishMyilsamy**
i have made some changes in home-style

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+body {
+    width: 80%;
+    max-width: 1280px;
```
Comment (puranik3) [inline css/home.css]: Why 1280? The requirements seems to be 1320px.

File: home.html [hunk 2]
```diff
@@ -16,5 +17,9 @@ <h1>About Git</h1>
         <a href="#">Twitter</a>
         <a href="#">About</a>
         <a href="#">My link</a>
+        <a href="#">Google</a>
```
Comment (puranik3) [inline home.html:20]: I think Twitter needs to be added as well.

**Review by puranik3 — CHANGES_REQUESTED** [review]
Comments are inline. The max-width needs change. Twitter link should be added.

File: home.html (hunk 2 above)
Comment (SathishMyilsamy) [reply home.html:20]: Twitter is not required

**Review by puranik3 — APPROVED** [review]

**Pull request merged**